nestjs/src/courses/team-distribution/dto/teams-query.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+
+export class TeamsQueryDto {
+  @ApiProperty()
+  @IsString()
+  public current: string;
+
+  @ApiProperty()
+  @IsString()
+  public pageSize: string;
```

Comment:
should it be string? both seems to be `number`

Reply:
fixed